### PR TITLE
Local development instructions

### PR DESCRIPTION
**Pull request by Sjors**
There may also be a Docker way to do this, but the following worked for me.

I had to make `/var/lib/config/isso.cfg` and `/var/lib/db/comments.db` configurable for this to work.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```sh
+flask db upgrade
+```
+
+Start the server:
+
+```sh
+export ISSO_CONFIG_PATH=$PWD
+export COMMENTS_DB_PATH=$PWD
+docker_boot.py & gunicorn patron:app
```
Comment (Sjors) [inline DEVELOPMENT.md]: I didn't check if `docker_boot.py` actually does anything.

**Comment by Sjors** [comment]
When running the tests one test fails with:

```
test_client = <FlaskClient <Flask 'app'>>, new_user = <User test>

    def test_scheduler(test_client, new_user):
        messages = []

        @email_dispatched.connect
        def suppressed_mail(message):
            messages.append(message)

>       message = messages[0]
E       IndexError: list index out of range
```

File: DEVELOPMENT.md (hunk 1 above)
Comment (JeffVandrewJr) [reply DEVELOPMENT.md]: `docker_boot.py` makes continual GET requests to ensure that settings load. Certain settings won't load until an initial GET request is received.

**Comment by JeffVandrewJr** [comment]
I appreciate this; thank you.

As you mention, it's a known issue that the scheduler testing fails (despite the scheduler currently working correctly in production), which should probably be fixed before any local development instructions are posted. Do you have any interest on working on fixing this one nagging test?

**Comment by Sjors** [comment]
I don't think fixing the test suite is a prerequisite. I'm not familiar enough with the framework to be of much using trying to repair a test yet. Hopefully these instructions lead to more developers who can help out in fixing it?

**Comment by JeffVandrewJr** [comment]
@Sjors Update: I fixed the tests so they all now pass. I still need to test the changes in this PR before I merge though.

**Comment by Sjors** [comment]
I rebased my PR on top of the latest master. The scheduler test takes 60 seconds which is annoying, but at least it passes.

**Comment by JeffVandrewJr** [comment]
@Sjors Yeah it has to wait for the scheduler; to make it go faster than a minute I'd have to refactor the scheduler stuff to accept a seconds input, which could be something for the future but not right now.

**Comment by JeffVandrewJr** [comment]
@Sjors I haven't forgotten about this; I've been busy helping some new users but should have this merged soon.

**Comment by JeffVandrewJr** [comment]
@Sjors Everything was good, so this is all merged!

Thanks again!

**Pull request merged**